README.md

```diff
@@ -32,9 +33,11 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+You'll need to sign up to trello and insert your Key & Token in the .env file.
```

Comment:
You haven't included useful details. The reader has to go and read the codebase to work out what the env vars should be called. And you also haven't mentioned the fact that they need `TRELLO_URL` and `TRELLO_BOARD_ID` as well.

Nicer would be to add all 4 variables with placeholder values to .env.template. Then it's easier for the reader to get set up.